_config.yml

```diff
@@ -34,12 +34,13 @@ header_pages:
   - about.markdown
   - events.markdown
 
-
 collections:
-  events:
-    output: true
-    future: true
-    permalink: /events/:name
+  posts:
+    permalink: /:categories/:title
```

Comment:
`/:categories` is the only way we can specify the path to be both language, and a specific subdirectory, events in our case. And `/:title` defaults to the file path. We can optionally use different file path names for different languages, OR specify the customised `:slug` in an individual post, which I did for German. `/:title` will dynamically pick either up

Reply:
I find the name "categories" confusing. Can't we rename it to "languages" ?

Reply:
custom FrontMatter are not available from within the config file. If we find another solution I would support it! This has two advantages, it allows language filtering, and also _post type.